Penteract/Assets/Scripts/FireTilesAnimationController.cpp

```diff
@@ -0,0 +1,35 @@
+#include "FireTilesAnimationController.h"
+
+#include "GameObject.h"
+
+EXPOSE_MEMBERS(FireTilesAnimationController) {    
+};
+
+GENERATE_BODY_IMPL(FireTilesAnimationController);
+
+void FireTilesAnimationController::Start() {
+	animation = GetOwner().GetComponent<ComponentAnimation>();
+
+	GameObject* owner = &GetOwner();
+
+	ComponentBoxCollider* bCollider = owner->GetComponent<ComponentBoxCollider>();
+	if (bCollider) {
+		bCollider->Disable();
```

Comment:
Is there a chance that the Box Collider isn't disabled when Starting?

Reply:
It happened that even though I saved a prefab with all colliders deactivated, when importing it to the engine to add it to the level, they were all activated. So Jordi and I made this change to make sure that they are all deactivated no matter what on start.